### ISSUE TITLE
Squidge-off winner playing a foul shot before time expires

### ISSUE THREAD

**Issue by hatj2** [etwa-rsc/rules#16]
What happens when the squidge-off winner plays a foul shot and time expires before they replay it under 2.3.2ii? 
Does time ending before the opponents have decided whether to make them replay or not change things? I feel that the opponents shouldn't be able to choose whether they are playing in 0 or 1, but I can't think at the moment, so I'll let someone else do the logic tree here.

**Comment by fluppeteer** [comment]
FWIW, I think the relevance of this is highly influenced by the current interpretation of 18.1. If we change the rules (#11) such that round 1 immediately follows the last Shot of the squidge-off winner that happens after time expires, as is currently proposed, then it doesn't matter how many times that player plays illegally.

**Comment by timhunt** [comment]
From Patrick

There are two categories of foul shot. "Standard" foul shots are those handled by Rule 23.2. "Out-of-sequence" foul shots are those handled by Rule 23.3.

For "standard" foul shots at the end of the timed period, I think the new wording of proposed Rule 18 works better than the old/current wording – indeed it's another reason for the proposed change.

For "out-of-sequence" foul shots at the end of the timed period, it is possible to imagine situations where the rules (past, present and future) are not clear.

I consider below two situations – one for standard foul shots, and one for an out-of-sequence situation.

-----

Case 1: Blue won squidge-off. Blue plays a foul shot. The timer bings before the next shot. If the foul shot is accepted, then it is Green to play in round zero. If the foul shot is rejected, then winks are replaced, Blue will replay a shot and proposed rule 18 states that this turn will end round zero. Green will play in round 1. That seems straightforward to me.

Harley’s question is what happens when the timer bings while Green/Yellow are still deciding whether to accept or reject the foul shot of Blue. This is the situation above. I don’t object to Green/Yellow having a choice that affects timing of rounds because they did not commit the foul shot! That said, I have indicated in another post that I would prefer time taken to decide whether foul shots are accepted/rejected not be counted as part of the game – which would make Harley’s situation impossible – but it is debatable whether this is what is currently stated in the rules.

-----

Case 2: Blue won squidge-off and it is Blue’s turn when the timer goes bing. Imagine the person playing Red forgets his/her/its colour and plays blue by mistake. It is easy if the foul shot is rejected – Blue would play a blue turn and end round zero. However, if the shot is accepted then the opponents have a choice of playing Green or Yellow. [Green is allowed as it follows a turn with blue; Yellow is allowed as it follows a shot played by the person playing Red.]. What round are we in? My common sense suggests if Green plays it should be in Round 1 (as it treats the previous shot as if by Blue), while if Yellow plays it should be in Round 0 (as it treats the previous shot as if by Red). But I don't think the rules specify anything for this particular situation.

For this particular case, you can take the point of view that it is very unlikely to happen in practice, and if it did a common sense approach would be used as I have described. Or you can take the view that we need to add to the wording of Rule 23.3(ii) to state what should happen.

**Comment by timhunt** [comment]
Regarding Patrick's case 2. We did not have exactly that scenario in the last Somerset Invitation, when Red played when it should have been blue's turn during round (but I think played a red). At that point, I had no clue what would happen, so had to get to the rules on my phone, and I was pleased with how clearly the rules are worded:

bq. 23.3.ii ... If this situation arises during the round limit period of the game (Rule 18), the counting of rounds should be such that no colour that is played in its correct order has more than one turn in any round. ...

So, my interpretatoin of the current rules are: If it was "blue to play ending round 0", then, even though they played before the clock went ping, red and yellow have already played in round zero. Therefore, if G/Y choose to accept the illegal shot as a red shot, then it is definitely yellow to play in 1.

**Comment by fluppeteer** [comment]
I think I agree with Timmy (yes, I do feel unwell). My interpretation of the "playing the wrong colour and choosing to follow the Colour who played rather than the Colour whose turn it was" rule is to treat it as though the Colour whose turn it was had implicitly passed, and so had the following Colour. That is, it was Blue's turn, 0 ended with an implicit pass by Blue, Green implicitly passed as well, Red played, and it's Yellow's turn in 1.

I'm slightly trying to avoid referring to "round 0", for what it's worth. That was one reason I preferred "the colour after the squidge-off winner starts round 1" over "the squidge-off winner ends 0", but I concede it may not quite be the same thing in this case, at least without careful wording.

Incidentally, while I believe clarification wording should be separated (we should define each rule only once - it's hard enough to stay mutually consistent even with that constraint), I get very nervous about "common sense", because it isn't. It's certainly easy to assume that everyone knows what we mean by something because everyone has spent years watching games of winks, and novices are likely not to realise why some situations are nonsensical. I'd rather ensure all bases are covered.

**Comment by hatj2** [comment]
I agree with Andrew - the point of the rules is that they should explain what happens in unusual scenarios so that when they do occur you can look it up and find it out simply. That's why the explanations are a good feature - we should include some for this bit, once we've worked out what the rule is/should be. [Aside: Croquet has a body of 'case law', the Official Rulings on the Laws of Croquet: https://www.croquet.org.uk/?p=games/association/laws/6th/orlac which is very dull indeed but presumably helps them no end when they revise the laws from time to time.]

> 17.3 The following instances are not counted as part of the game: time elapsing while a wink or winks are lost; time taken to correct the outcome of a foul shot; time taken for an umpire to arrive if one is called for; time taken over a judgement determining the status of winks (e.g. squopped or unsquopped); time taken over discussions concerning the interpretation of a rule; time taken for discussions with the umpire.

Do we think that 'Time taken to correct theoutcome of a foul shot' includes the second (and third) attempt? Is that what we do for foul shots in time now or do we re-start the clock when the shot has been set up again? I think we ought to clarify this within 7.3 anyway. But the interpretation of this has clear relevance for our question; if the time between the end of the foul shot and the end of that turn doesn't count, then this isn't an issue. (What happens if the second shot is a pot?)

I'm too confused to think about (2) at this stage - I need to sit down and work it out with a pencil...

**Comment by fluppeteer** [comment]
That's a Friday evening can of worms. With the proviso that I think there's already a statement that umpire involvement stops the clock, the rest of this still applies even with just the normal players.

There are two, possibly orthogonal, issues:

1. Red plays a dubious shot, Yellow and Green argue about whether it was legal. Whose Shot/Turn is it while they argue? (If they spend a long time over doing so, can Red stop the clock? Can anyone practise?) Does it depend on the outcome of the decision?

2. Having decided that the shot was illegal, Yellow and Green argue about whether to accept it or ask for it to be replayed. Whose turn is it while they decide? Does it depend on the outcome of the decision?

Related: Time to determine whether a Colour can play should be part of that Colour's turn, I would argue (with the clock stopped for umpire involvement or by mutual agreement). What happens if there's a dispute about the number of squop-up turns, though?

**Comment by hatj2** [comment]
> Regarding Patrick's case 2. We did not have exactly that scenario in the last Somerset Invitation, when Red played when it should have been blue's turn during round (but I think played a red). At that point, I had no clue what would happen, so had to get to the rules on my phone, and I was pleased with how clearly the rules are worded:
> bq. 23.3.ii ... If this situation arises during the round limit period of the game (Rule 18), the counting of rounds should be such that no colour that is played in its correct order has more than one turn in any round. ...
> So, my interpretatoin of the current rules are: If it was "blue to play ending round 0", then, even though they played before the clock went ping, red and yellow have already played in round zero. Therefore, if G/Y choose to accept the illegal shot as a red shot, then it is definitely yellow to play in 1.

Patrick's proposed reworded 18 disagrees with this: "after the turn ends of the SoW, 5 rounds are played"; so if you accept the illegal shot as a red shot, there hasn't been a Blue turn so you can't be in round 1 yet. I don't think this contradicts 23.3ii; 'round 0' is Not A Thing according to the rules, it's just a construct players have to help them count. Before round 1 there just some turns.
